### PR TITLE
feat(docs): add example to clip_vector_norm doc

### LINKED ISSUE #26481: clip_vector_norm

### PR DESCRIPTION
**Pull request by monsterdevgit**
## PR Description:
Added an example to the clip_vector_norm docstring. The added example conforms with Point 7 of Ivy's Docstring examples. Which states: 
> passes in ivy.Container instances for multiple arguments

## Related Issue:

Close #26481

## Checklist:
- [ ] Added a new function
- [ ] Added relevant tests for the new function
- [x] Successfully ran tests, and all are passing
- [x] pre-commit did not fail on any check
- [x] Followed the steps provided for this contribution

### CODE REVIEW

**Comment by monsterdevgit** [comment]
I have not received a response for this PR.
Am new to open source contributing, so will like some guidance please.

File: ivy/functional/ivy/test.py
Comment (umairjavaid) [inline ivy/functional/ivy/test.py:1]: Kindly remove this file. This is not needed.

**Review by umairjavaid — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/test.py
Comment (monsterdevgit) [reply ivy/functional/ivy/test.py:1]: Done. I don't know how it got uploaded.

**Comment by aparajith21** [comment]
Hey, thanks for contributing. LGTM!

**Pull request merged**